Wikipedia/UI-V5/WMFRecentPagesDataSource.m

```diff
@@ -36,6 +36,7 @@ - (nonnull instancetype)initWithRecentPagesList:(MWKHistoryList*)recentPages {
             cell.descriptionText       = article.entityDescription;
             cell.image                 = [article bestThumbnailImage];
             cell.summaryAttributedText = [article summaryHTMLWithoutLinks];
+            cell.title                 = article.title;
         };
```

Comment:
Since I have to pass title, I guess I could remove cell.titleText and get the article title from cell.title...